python_scripts/parameter_tuning_sol_03.py

```diff
@@ -69,8 +74,8 @@
 from sklearn.model_selection import RandomizedSearchCV
 
 param_distributions = {
-    "kneighborsregressor__n_neighbors": np.logspace(0, 3, num=10).astype(
-        np.int32
+    "kneighborsregressor__n_neighbors": (
+        np.logspace(0, 3, num=10).astype(np.int32)
```

Comment:
This should fix the CI:
```suggestion
    "kneighborsregressor__n_neighbors": np.logspace(0, 3, num=10).astype(
        np.int32
```